test/extended/openstack/loadbalancers.go

```diff
@@ -208,6 +209,85 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb] The Openstack
 				e2e.Logf("Pod %s successfully accessed through svc loadbalancer on ip %s and port %d", podName, fip, svcPort)
 			}
 		})
+
+		// https://issues.redhat.com/browse/OCPBUGS-2350
+		g.It(fmt.Sprintf("should apply lb-method on UDP %s LoadBalancer when an UDP svc with monitors and ETP:Local is created on Openshift", lbProviderUnderTest), func() {
+
+			skipIfNotLbProvider(lbProviderUnderTest, cloudProviderConfig)
+
+			g.By("Creating Openshift deployment")
+			labels := map[string]string{"app": "udp-lb-etplocal-dep"}
+			testDeployment := createTestDeployment("udp-lb-etplocal-dep", labels, 2, v1.ProtocolUDP, 8081)
+			deployment, err := clientSet.AppsV1().Deployments(oc.Namespace()).Create(context.TODO(),
+				testDeployment, metav1.CreateOptions{})
+			o.Expect(err).NotTo(o.HaveOccurred())
+			err = e2edeployment.WaitForDeploymentComplete(clientSet, deployment)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("Creating Openshift LoadBalancer Service with monitors and ETP:Local")
+			svcName := "udp-lb-etplocal-svc"
+			svcPort := int32(8082)
+			monitorDelay := 10
+			monitorTimeout := 20
+			monitorMaxRetries := 2
+			jig := e2eservice.NewTestJig(clientSet, oc.Namespace(), svcName)
+			jig.Labels = labels
+			svc, err := jig.CreateLoadBalancerService(5*time.Minute, func(svc *v1.Service) {
+				svc.Spec.Ports = []v1.ServicePort{{Protocol: v1.ProtocolUDP, Port: svcPort, TargetPort: intstr.FromInt(8081)}}
+				svc.Spec.Selector = labels
+				svc.SetAnnotations(map[string]string{
+					"loadbalancer.openstack.org/enable-health-monitor":      "true",
+					"loadbalancer.openstack.org/health-monitor-delay":       fmt.Sprintf("%d", monitorDelay),
+					"loadbalancer.openstack.org/health-monitor-timeout":     fmt.Sprintf("%d", monitorTimeout),
+					"loadbalancer.openstack.org/health-monitor-max-retries": fmt.Sprintf("%d", monitorMaxRetries),
+				})
+				svc.Spec.ExternalTrafficPolicy = v1.ServiceExternalTrafficPolicyTypeLocal
+			})
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("Checks from openshift perspective")
+			loadBalancerId := svc.GetAnnotations()["loadbalancer.openstack.org/load-balancer-id"]
+			o.Expect(loadBalancerId).ShouldNot(o.BeEmpty(), "load-balancer-id annotation missing")
+			o.Expect(svc.Status.LoadBalancer.Ingress).ShouldNot(o.BeEmpty(), "svc.Status.LoadBalancer.Ingress should not be empty")
```

Comment:
This is a bit redundant, it's checked by WaitForLoadBalancer: https://github.com/openshift/kubernetes/blob/020cf2d7aa2011147a89c9bd4063c8237b71a08f/test/e2e/framework/service/jig.go#L581-L586

Reply:
good catch! I'm removing it